code/vocabulary.py

```diff
@@ -38,92 +95,40 @@
 ]
 
 # Default words that need to be remapped.
-_word_map_defaults = {
+default_word_map = {
     # E.g:
     # "cash": "cache",
+    # "centre": "center",
 }
+default_word_map.update({word.lower(): word for word in capitalize})
 
-# Default words that should be added to Talon's vocabulary.
-_simple_vocab_default = ["nmap", "admin", "Cisco", "Citrix", "VPN", "DNS", "Minecraft"]
-
-# Defaults for different pronounciations of words that need to be added to
-# Talon's vocabulary.
-_mapping_vocab_default = {
-    "N map": "nmap",
-    "under documented": "under-documented",
-}
-
-
-mod = Module()
-
-
-@mod.capture(rule="{user.vocabulary}")
-def vocabulary(m) -> str:
-    return m.vocabulary
-
-
-@mod.capture(rule="(<user.vocabulary> | <word>)")
-def word(m) -> str:
-    try:
-        return m.vocabulary
-    except AttributeError:
-        # TODO: if the word is both a regular word AND user.vocabulary, then in
-        # principle it may parse as <word> instead; we ought to pass it through
-        # mapping_vocabulary to be sure. But we should be doing that in
-        # user.text, below, too.
-        words = actions.dictate.replace_words(actions.dictate.parse_words(m.word))
-        assert len(words) == 1
-        return words[0]
-
-
-punctuation = set(".,-!?;:")
-
-
-@mod.capture(rule="(<user.vocabulary> | <phrase>)+")
-def text(m) -> str:
-    words = []
-    for item in m:
-        if isinstance(item, grammar.vm.Phrase):
-            words.extend(
-                actions.dictate.replace_words(actions.dictate.parse_words(item))
-            )
-        else:
-            words.extend(item.split(" "))
-
-    result = ""
-    for i, word in enumerate(words):
-        if i > 0 and word not in punctuation and words[i - 1][-1] not in ("/-("):
-            result += " "
-        result += word
-    return result
-
-
-mod.list("vocabulary", desc="user vocabulary")
-
-ctx = Context()
-
-
-_default_word_map = {}
-_default_word_map.update(_word_map_defaults)
-_default_word_map.update({word.lower(): word for word in _capitalize_defaults})
 # "dictate.word_map" is used by `actions.dictate.replace_words` to rewrite words
 # Talon recognized. Entries in word_map don't change the priority with which
 # Talon recognizes some words over others.
 bind_word_map_to_csv(
     "words_to_replace.csv",
     csv_headers=("Replacement", "Original"),
-    default_values=_default_word_map,
+    default_values=default_word_map,
 )
 
-_default_vocabulary = {}
-_default_vocabulary.update({word: word for word in _simple_vocab_default})
-_default_vocabulary.update(_mapping_vocab_default)
+
+# Default words that should be added to Talon's vocabulary.
+simple_vocabulary = ["admin", "LCD", "VPN", "DNS", "USB", "FAQ", "PhD", "Minecraft"]
```

Comment:
Shouldnt change these defaults. Really not sure why they are in python at all instead of only in the csv files.

Reply:
The defaults are only used when someone doesn't have a corresponding csv file, and they are used to create the initial csv file, so I think it might be sensible to have defaults that are more likely to be useful for most people. I've reverted these changes, however.

Reply:
that makes sense